Exercises/operators-and-type-conversions/operators-and-type-conversions.js

```diff
@@ -0,0 +1,21 @@
+// Task #1
+a = 10;
+console.log(parseInt(a + '0'));
+
+// Task #2
+const stringValue = `Amazon annual revenue of US$232.887 billion`;
+let r = new RegExp(/\d+.\d+/);
+let annualRevenue = +stringValue.match(r);
```

Comment:
cast with `Number`

Reply:
Sorry, did you mean to use Number(object) instead of +stringValue.match() with RegExp?
but Number (stringValue) gives NaN.
Could you please provide some explanations?
Thank you)